spec/features/user_browse_items_spec.rb

```diff
@@ -0,0 +1,28 @@
+# As an unauthenticated user...
+#When I visit the home page...
+#Then I expect to see a list of available spooky items (root to items_path)
+
+require 'rails_helper'
+
+feature 'browse items view' do
+  before(:each) do
+    Item.create(title: "Eat Cheese and Die",
+                description: "The spookiest grilled cheese.",
+                price: 50.65)
+    Item.create(title: "It Came From Beneath the Sink",
+                description: "You don't want to know.",
+                price: 200.99)
+    visit root_path
+  end
+
+  scenario 'view items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.